src/blueprint/into_schema.rs

```diff
@@ -177,20 +181,25 @@ fn to_type(def: &Definition) -> dynamic::Type {
   }
 }
 
-fn create(blueprint: &Blueprint) -> SchemaBuilder {
+#[derive(Copy, Clone, Debug)]
+pub struct SchemaModifiers {
+  pub no_resolver: bool,
+}
+
+pub(crate) fn create(blueprint: &Blueprint, modifiers: Option<SchemaModifiers>) -> SchemaBuilder {
   let query = blueprint.query();
   let mutation = blueprint.mutation();
   let mut schema = dynamic::Schema::build(query.as_str(), mutation.as_deref(), None);
 
   for def in blueprint.definitions.iter() {
-    schema = schema.register(to_type(def));
+    schema = schema.register(to_type(def, modifiers.map(|a| a.no_resolver).unwrap_or(false)));
   }
 
   schema
 }
 
 impl From<&Blueprint> for SchemaBuilder {
   fn from(blueprint: &Blueprint) -> Self {
-    create(blueprint)
+    create(blueprint, None)
```

Comment:
Why are we making a change here?

Reply:
I'm reusing this method to create a validation schema that doesn't make actual calls. The method accepts an additional `Option<SchemaModifiers>` to modify the resulting schema. Normal behavior is then obtained by passing `None`.